index.js

```diff
@@ -223,7 +223,7 @@ export default class Emittery {
 
 		// eslint-disable-next-line n/prefer-global/process
```

Comment:
It would be clearer to change `typeof globalThis.process !== 'object'` to `typeof globalThis.process?.env !== 'object'`.

It also needs a code comment to quickly explain why we don't just check `globalThis.process`.

Reply:
Thanks @sindresorhus , addressed in 75bea4b5853c4d82e032bd5048e98f7850316f7b